### PR TITLE
feat(PodcastAddict): new logo and add cover art

### PR DESCRIPTION
**Pull request by Makogai**
Updated logo

### CODE REVIEW

**Comment by Makogai** [comment]
Bruh what

**Comment by kyrie25** [comment]
> Bruh what

You need to bump version + run your codes through Prettier

**Comment by Slowlife01** [comment]
Provide proof

**Comment by Makogai** [comment]
PROOF: 
image
image

File: websites/P/PodcastAddict/presence.ts [hunk 1]
```diff
@@ -22,6 +22,9 @@ presence.on("UpdateData", async () => {
   else if (pathname.startsWith("/podcast")) {
     presenceData.details = "Viewing:";
     presenceData.state = document.querySelector(".caption").textContent;
+    presenceData.largeImageKey = document.querySelector<HTMLImageElement>(
+      ".headerThumbnail img"
+    ).src;
```
Comment (Bas950) [inline websites/P/PodcastAddict/presence.ts:27]: https://discord.com/channels/493130730549805057/607524579874832446/928096461994987540

**Review by Bas950 — APPROVED** [review]

**Review by Slowlife01 — CHANGES_REQUESTED** [review]
Update your fork

**Comment by Makogai** [comment]
> Update your fork

Will be done soon

**Review by EncryptedDev — APPROVED** [review]

**Comment by EncryptedDev** [comment]
Inactive

**Pull request closed without merging**